libs/pageserver_api/src/key.rs

```diff
@@ -21,9 +23,78 @@ pub struct Key {
     pub field6: u32,
 }
 
+/// The storage key size.
 pub const KEY_SIZE: usize = 18;
 
+/// The metadata key size. 2B fewer than the storage key size because field2 is not fully utilized.
+/// See [`Key::to_i128`] for more information on the encoding.
+pub const METADATA_KEY_SIZE: usize = 16;
+
+/// The key prefix start range for the metadata keys. All keys with the first byte >= 0x80 is a metadata key.
+pub const METADATA_KEY_BEGIN_PREFIX: u8 = 0x80;
+
+/// The key prefix of AUX file keys.
+pub const AUX_KEY_PREFIX: u8 = 0x90;
```

Comment:
Let's think ahead a bit, since these field1 values are very permanent choices -- maybe we should plan for our main database metadata (relation sizes etc) to be 0x81, and make aux files 0x82?  This is obviously all quite arbitrary.

The reason I'd like to pack these close together is in case we want to do something else in future with the higher part of the field1 space.

Reply:
fixed in https://github.com/neondatabase/neon/pull/7401/commits/6fc849b7442602b5fbd061bb48d34a624889b21d